src/utils.ts

```diff
@@ -545,44 +549,38 @@ export interface ClientMetadataOptions {
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const NODE_DRIVER_VERSION = require('../package.json').version;
 
-export function makeClientMetadata(options?: ClientMetadataOptions): ClientMetadata {
-  options = options ?? {};
+export function makeClientMetadata(
+  options: Pick<MongoOptions, 'appName' | 'driverInfo'>
+): ClientMetadata {
+  const name = options.driverInfo.name ? `nodejs|${options.driverInfo.name}` : 'nodejs';
+  const version = options.driverInfo.version
+    ? `${NODE_DRIVER_VERSION}|${options.driverInfo.version}`
+    : NODE_DRIVER_VERSION;
+  const platform = options.driverInfo.platform
+    ? `Node.js ${process.version}, ${os.endianness()} (unified)|${options.driverInfo.platform}`
```

Comment:
What does `(unified)` mean?

Reply:
https://github.com/mongodb/node-mongodb-native/commit/ade9a62301abfdb093fbbc2ff37c0a2e6776f977

We used to report whether the unified or legacy topology was being used, I think we can remove this now.

Reply:
alex is on board for this as well

Reply:
addressed